src/sentry/api/bases/organization_events.py

```diff
@@ -33,13 +34,34 @@ def has_feature(self, organization, request):
 
     def get_snuba_filter(self, request, organization, params=None):
         if params is None:
-            params = self.get_filter_params(request, organization)
+            params = self.get_snuba_params(request, organization)
         query = request.GET.get("query")
         try:
             return get_filter(query, params)
         except InvalidSearchQuery as e:
             raise ParseError(detail=six.text_type(e))
 
+    def get_snuba_params(self, request, organization):
+        with sentry_sdk.start_span(op="discover.endpoint", description="filter_params") as span:
+            if len(request.GET.getlist("field")) > MAX_FIELDS:
+                raise ParseError(
+                    detail="You can view up to {0} fields at a time. Please delete some and try again.".format(
+                        MAX_FIELDS
+                    )
+                )
+
+            span.set_tag("organization", organization)
```

Comment:
If the organization is already an event level tag do we need it on the span as well?